packages/snap-preact/src/setupEvents.ts

```diff
@@ -48,5 +48,40 @@ export const setupEvents = () => {
 		await next();
 	});
 
+	type ControllerUpdateRecsData = {
+		controllerIds?: (string | RegExp)[];
+	};
+
+	eventManager.on('controller/updateRecs', async (data: ControllerUpdateRecsData, next: Next) => {
```

Comment:
I wonder if we should make this more generic ... maybe `controller/search` or `controller/update` - `update` doesn't sound like what it is actually doing, I'm not sure which is better... 

Ideas:
```
eventManager.on('controller/updateRecs');  // current
eventManager.on('controller/search');  // not restrictive enough, but generic for future usage
eventManager.on('controller/searchRecs');  // similar to search but with additional filtering
eventManager.on('controller/recommend/search');  // similar to search but with additional filtering
```

Just thoughts, I'd like to have a good structure and naming convention for these global events.